src/main/kotlin/no/skatteetaten/aurora/boober/utils/DeploymentConfigUtils.kt

```diff
@@ -0,0 +1,33 @@
+package no.skatteetaten.aurora.boober.utils
+
+import com.fasterxml.jackson.databind.node.ArrayNode
+import io.fabric8.openshift.api.model.DeploymentConfig
+import no.skatteetaten.aurora.boober.service.openshift.OpenShiftResponse
+
+data class ImageInformation(val lastTriggeredImage: String, val imageStreamName: String, val imageStreamTag: String)
+
+fun DeploymentConfig.findImageInformation(): ImageInformation? {
+    val imageChangeParams = this.spec?.triggers?.get(0)?.imageChangeParams
+    val lastTriggeredImage = imageChangeParams?.lastTriggeredImage ?: ""
+    return imageChangeParams?.from?.name?.let {
+        val (isName, tag) = it.split(':')
+        ImageInformation(lastTriggeredImage, isName, tag)
+    }
+}
+
+fun DeploymentConfig.didImportImage(response: OpenShiftResponse): Boolean {
+    val body = response.responseBody ?: return true
```

Comment:
Blir disse "guardene" rett. Skal vi si at vi lykkes med å importere et image hvis man ikke har en responseBody?

Reply:
Bra observert :)
Det er nok jeg som har sett for mye på den gamle implementasjonen hvor det stod slik:
```
val body = response.responseBody ?: return true
```
Men det er ikke nødvendigvis slik vi vil ha det.

@oyvinder vet vi om noen tilfeller hvor dette kan skje?